services/ui/ws/window_tree.cc

```diff
@@ -1566,6 +1566,14 @@ void WindowTree::SetWindowVisibility(uint32_t change_id,
       SetWindowVisibility(ClientWindowId(transport_window_id), visible));
 }
 
+void WindowTree::HideNativeWindow(Id transport_window_id) {
```

Comment:
it goes here from WindowTreeHostMus::HideImpl. In internal window mode, it will produce a crash